spark_log_parser/loaders/__init__.py

```diff
@@ -46,16 +46,12 @@ class FileChunkStreamWrapper:
     Note #2 - once `chunks` is consumed, calls to read/lines will return empty byte buffers
     """
 
-    total_size: int = 0
-
-    _maximum_allowed_size: ArchiveExtractionThresholds
-    _trailing_data: bytes = b''
-    _chunks: Iterator[bytes]
-
     def __init__(self, chunks: Iterator[bytes], maximum_file_size: int = None):
         assert chunks
-        self._chunks = chunks
+        self.total_size: int = 0
+        self._chunks: Iterator[bytes] = chunks
```

Comment:
probably want to keep the type hints

Reply:
oo yep, missed a few

Reply:
done :+1:

Reply:
Without the history that was wiped away by the rebase/force-push I don't understand what Wes was talking about.

Reply:
I missed adding type hints to some attributes just below these!